src/modules/task/dto/update-task.dto.ts

```diff
@@ -0,0 +1,27 @@
+import { Type } from 'class-transformer';
+import { IsEnum, IsInt, IsNumber, IsOptional, IsPositive, IsString, MinLength } from 'class-validator';
+import { Severity } from '../task.types';
+
+export class UpdateTaskParamsDto {
+  @IsNumber()
+  @IsPositive()
+  @IsInt()
+  @Type(() => Number)
+  id: number;
+}
+
+export class UpdateTaskBodyDto {
+  @IsOptional()
+  @IsString()
+  @MinLength(1)
+  title: string;
+
+  @IsOptional()
+  @IsString()
+  @MinLength(1)
+  description: string;
+
+  @IsOptional()
+  @IsEnum(Severity)
+  severity: Severity;
```

Comment:
```suggestion
  @IsOptional()
  @IsString()
  @MinLength(1)
  title?: string;

  @IsOptional()
  @IsString()
  @MinLength(1)
  description?: string;

  @IsOptional()
  @IsEnum(Severity)
  severity?: Severity;
```
все поля опциональные, значит их может быть, а тайпскрипту ты этого не сообщил

Reply:
их может **НЕ** быть*